powertools-cloudformation/pom.xml

```diff
@@ -32,28 +32,6 @@
         A suite of utilities for AWS Lambda Functions that makes tracing with AWS X-Ray, structured logging and creating
         custom metrics asynchronously easier.
     </description>
-    <url>https://aws.amazon.com/lambda/</url>
```

Comment:
Great to see this get cleaned up! So much noise and duplication that makes it hard to see the woods for the trees.

Reply:
We should also try run a release locally, without credentials, to make sure that we get to the point where we are pushing to maven (and failing!). I think this should all inherit down, but it would be nice to make sure it does now rather than finding out later :D